Makefile

```diff
@@ -0,0 +1,75 @@
+BASE_IMG_NAME = bedrock_base
+DEV_IMG_NAME = bedrock_dev
+FINAL_IMG_NAME = bedrock_dev_final
+
+default:
+	@echo "You need to specify a subcommand."
+	@exit 1
+
+help:
+	@echo "build           - build docker containers for dev"
+	@echo "run             - docker-compose up the entire system for dev"
+	@echo ""
+	@echo "clean           - remove all build, test, coverage and Python artifacts"
```

Comment:
Since this removes all `build` artifacts maybe it should also remove docker images?